test/unit/object/test_comment.py

```diff
@@ -0,0 +1,95 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+import json
+import pytest
+
+from boxsdk.object.comment import Comment
+
+
+# pylint:disable=protected-access
+# pylint:disable=redefined-outer-name
+
+@pytest.mark.parametrize(
+    'message_type, message',
+    [
+        # Test case for plain message
+        (
+            'message',
+            'Hello there!'
+        ),
+
+        # Test case for tagged message
+        (
+            'tagged_message',
+            '@[22222:Test User] Hi!'
+        )
+    ]
+)
+def test_reply(test_comment, mock_box_session, message_type, message):
+    expected_url = mock_box_session.get_url('comments')
+    expected_data = {
+        message_type: message,
+        'item': {
+            'type': 'comment',
+            'id': test_comment.object_id
+        }
+    }
+    mock_box_session.post.return_value.json.return_value = {
+        'type': 'comment',
+        'id': '12345',
+        message_type: message
+    }
+    reply_comment = test_comment.reply(message)
+    mock_box_session.post.assert_called_once_with(expected_url, data=json.dumps(expected_data))
+    assert reply_comment.object_id == '12345'
+
+
+@pytest.mark.parametrize(
```

Comment:
This is copy pasta from the previous test - can we either put this into a fixture or just pull out the parametrize call, like

    comment_parameters = @pytest.mark.parametrize(...)

    @comment_parameters
    def test_edit(...)